OpenRA.Mods.RA/Traits/Disguise.cs

```diff
@@ -49,10 +49,10 @@ public Player Owner
 		{
 			get
 			{
-				if (disguise.Disguised)
-					return self.Owner == self.World.LocalPlayer ? self.Owner : disguise.AsPlayer;
+				if (!disguise.Disguised || self.Owner.IsAlliedWith(self.World.RenderPlayer))
```

Comment:
This also uses RenderPlayer, to determine radar colors: https://github.com/deniz1a/OpenRA/blob/disguise-owner/OpenRA.Mods.RA/Traits/Disguise.cs#L125

But is that not a problem because it doesn't affect game state?